.travis.yml

```diff
@@ -3,9 +3,8 @@ node_js:
   - "4.2.4"
 env:
   - CMD="sass:compile"
-  - CMD="sass-lint"
-  - CMD="js-lint"
-#  - CMD="test:visual"
+  - CMD="lint:sass"
```

Comment:
makes sense. in case we need to add some other kind of linter. lint:magic
better naming convention 👍